tests/pytorch-community/base.libsonnet

```diff
@@ -0,0 +1,30 @@
+local base = import '../../templates/base.libsonnet';
+local gpus = import '../../templates/gpus.libsonnet';
+
+{
+  PytorchTest:: base.BaseTest {
+    frameworkPrefix: 'pt-1.4', # TODO: Change to match your version.
+    image: 'gcr.io/xl-ml-test/pytorch-community', # Change to your image.
+    imageTag: 'latest',
+    publisherImage: 'gcr.io/xl-ml-test/publisher:latest',
```

Comment:
Change this to your project's version of the publisher docker image.

Make this with:
`gcloud builds submit --config images/publisher/cloudbuild.yaml`

Full details on this part [here](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/doc/tutorials/test-templates-automated.md#automating-tests-with-cronjobs)